SABT_MainUnit/dictionary.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * dictionary.h
+ *
+ * Created: 6/18/2015 8:45:21 AM
+ *  Author: Susan Zuo(tzuo)
+ */ 
+
+
+#ifndef DICTIONARY_H_
+#define DICTIONARY_H_
+
+typedef struct dictionary dictionary_t;
+/**
+ *  a struct to store words loaded from SD card

```

Comment:
Let us know if this is ready to use/how to use, or if it's a work in progress what your plan is.